docs/content/modules/phase_field/introduction/PhaseFieldEquations.md

```diff
@@ -0,0 +1,220 @@
+
+To demonstrate this approach, we focus on a simple system with a single conserved concentration $c$ that varies from -1 to 1. The free energy in this case is
+
+$$
+f_{loc} = \frac{1}{4}(1 + c)^2(1 - c)^2
+$$
+
+and its derivatives are
+
+$$
+\begin{eqnarray}
+  \frac{\partial f_{loc}}{\partial c} &=& c(c^2 - 1) \\
+  \frac{\partial^2 f_{loc}}{\partial c^2} &=& 3 c^2 - 1 \\
+  \frac{\partial^3 f_{loc}}{\partial c^3} &=& 6 c.
+\end{eqnarray}
+$$
+
+The second and third derivatives would be required to use the direct solution method (via [`CahnHilliard`](/Kernels/CahnHilliard.md)) and the first and second derivatives would be required to use the split solution method (via [`SplitCHParsed`](/Kernels/SplitCHParsed.md)). This model has been implemented in the [`MathFreeEnergy`](/Materials/MathFreeEnergy.md) material found in the phase field module of MOOSE, inheriting from `DerivativeFunctionMaterialBase`. The code from [`MathFreeEnergy`](/Materials/MathFreeEnergy.md) is shown below:
+
+```cpp
+Real
+MathFreeEnergy::computeF()
+{
+  return 1.0/4.0*(1.0 + _c[_qp])*(1.0 + _c[_qp])*(1.0 - _c[_qp])*(1.0 - _c[_qp]);
+}
+
+Real
+MathFreeEnergy::computeDF(unsigned int j_var)
+{
+    return _c[_qp]*(_c[_qp]*_c[_qp] - 1.0);
+}
+
+Real
+MathFreeEnergy::computeD2F(unsigned int j_var, unsigned int k_var)
+{
+    return 3*_c[_qp]*_c[_qp] - 1.0;
+}
+
+Real
+MathFreeEnergy::computeD3F(unsigned int j_var, unsigned int k_var, unsigned int l_var)
+{
+    return 6*_c[_qp];
+}
+```
+
+An alternative to writing your own free energy materials is to take advantage of the [`DerivativeParsedMaterial`](FunctionMaterials), where the free energy is entered in the input file and all required derivatives are taken automatically. This approach is highly encouraged, as it drastically simplifies model development.
+
+
+## See also
+
+* [Phase Field FAQ](FAQ) - Frequently asked questions for MOOSE phase-field models.
+* [Function Materials](FunctionMaterials) - The key component in the modular free energy phase field modeling approach. This page lists the available function materials and explains how to define a free energy function and combine multiple free energy contributions (including elastic energy) into a total free energy.
+* [Function Material Kernels](FunctionMaterialKernels) - Kernels which utilize free energy densities provides by Function Material. These are the recommended phase field kernels.
+* [ExpressionBuilder](ExpressionBuilder) - Use automatic differentiation with Free energies defined in the C++ code.
+* [Multi Phase Models](MultiPhaseModels) - Combine multiple single phase free energies into multiphase field models using these tools.
+
+Free energies can also be combined with the deformation energy calculated using the Tensor Mechanics module.
```

Comment:
I think you should make this a bullet as well, it is just hanging down here and looks lonely.